spec/e2e/support/target-blank.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @typedef SimplifiedLink
+ *
+ * @prop {string} innerText
+ * @prop {string} href
+ * @prop {string} className
+ * @prop {string} target
+ */
+
+/**
+ * @param {import('puppeteer').Page} page */
+ * @return {Promise<SimplifiedLink[]>}
+ */
```

Comment:
Oops, my fault from the suggestion:

```suggestion
/**
 * @param {import('puppeteer').Page} page
 * @return {Promise<SimplifiedLink[]>}
 */
```

Reply:
It's all good, I caught it and fixed it in 0008fef2fb6d4a067f3685472ec53bd441cec3b7